src/menu.js

```diff
@@ -1,198 +1,65 @@
-      <div class="card-body">
-        <h5 class="card-title">Pounded yam and Egusi</h5>
-        <p class="card-text">This is a wider card with supporting text below as a natural lead-in to additional content. This content is a little bit longer.</p>
-        <p class="card-text"><small class="text-muted">Last updated 3 mins ago</small></p>
-      </div>
-    </div>
-  </div>
-  <div class="col-lg-4 col-md-6 col-sm-12 col-xl-4 shadow p-3">
-    <div class="card mb-3">
-      <img class="card-img-top" src="https://answersafrica.com/wp-content/uploads/2016/03/nigerian-rice-and-stew.jpg" alt="Card image cap">
-      <div class="card-body">
-        <h5 class="card-title">Rice and Stew</h5>
-        <p class="card-text">This is a wider card with supporting text below as a natural lead-in to additional content. This content is a little bit longer.</p>
-        <p class="card-text"><small class="text-muted">Last updated 3 mins ago</small></p>
-      </div>
-    </div>
-  </div>
-  <div class="col-lg-4 col-md-6 col-sm-12 col-xl-4 shadow p-3">
-    <div class="card mb-3">
-      <img class="card-img-top" src="https://res.6chcdn.feednews.com/assets/v2/e1021625a9edd4db53e7fabeb15cf20c?quality=uhq&resize=720&watermark=false" alt="Card image cap">
-      <div class="card-body">
-        <h5 class="card-title">Boiled plantain</h5>
-        <p class="card-text">This is a wider card with supporting text below as a natural lead-in to additional content. This content is a little bit longer.</p>
-        <p class="card-text"><small class="text-muted">Last updated 3 mins ago</small></p>
-      </div>
-    </div>
-  </div>
-  <div class="col-lg-4 col-md-6 col-sm-12 col-xl-4 shadow p-3">
-    <div class="card mb-3">
-      <img class="card-img-top" src="https://miro.medium.com/max/2800/0*Ye-z3Cbc7XQYeJq6.jpg" alt="Card image cap">
-      <div class="card-body">
-        <h5 class="card-title">Hot Okpa</h5>
-        <p class="card-text">This is a wider card with supporting text below as a natural lead-in to additional content. This content is a little bit longer.</p>
-        <p class="card-text"><small class="text-muted">Last updated 3 mins ago</small></p>
-      </div>
-    </div>
-  </div>
-</div>
+  // mainContainer
+  let mainContainerItems = [];
+  for(let i = 0; i < 6; i++) {
+    let div = document.createElement('div');
+    div.setAttribute('class', 'menu-items');
 
+    // Create a new image HTML element
+    const img = new Image();
+    
+    switch(i) {
+      case 0:
+        img.src = makChangImg;
+        div.appendChild(img);
+        break;
+    
+      case 1:
+        img.src = kimbapImg;
+        div.appendChild(img);
+        break;
+    
+      case 2:
+        img.src = spicyRadishImg;
+        div.appendChild(img);
+        break;
+    
+      case 3:
+        img.src = bibimbapImg;
+        div.appendChild(img);
+        break;
 
```

Comment:
Let separate the DOM from application logic, you can create another file for login or create another module/function for it.

![image](https://user-images.githubusercontent.com/31897434/73420341-70e18b80-4322-11ea-840a-fa14e04b592b.png)

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59623090/